docs/process.md

```diff
@@ -2075,6 +2075,9 @@ The following options are available:
 `runAsUser: '<uid>'`
 : Specifies the user ID with which to run the container. Shortcut for the `securityContext` option.
 
+`schedulerName: '<name>'`
+: Specifies which [scheduler](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/#specify-schedulers-for-pods) is used to schedule the container. 
```

Comment:
Which K8s version was this feature introduced? (worth to mention)

Reply:
It is quite old, at least as old as many other options for which we don't mention the minimum version

Reply:
Ok, was not aware of that. then we can assume it's ok